converter/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -101,7 +101,8 @@
 	CPC_PLUS_TOO_FEW_QUALITY_MEASURE_CATEGORY(64,
 			"CPC+ Submissions must have at least %d of the following %s measures: %s.", true),
 	CPC_PLUS_TOO_FEW_QUALITY_MEASURES(65,
-			"CPC+ Submissions must have at least %d of the following measures: %s.", true);
+			"CPC+ Submissions must have at least %d of the following measures: %s.", true),
+	CPC_PLUS_SUBMISSION_ENDED(66, "CPC+ Submission is after the end date %s.", true);
 
```

Comment:
I believe that I also use `ErrorCode 66`. Just a heads up.

Reply:
Whoever merges last will have their tests fail. ErrorCodeTest ensures this.